testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -728,36 +730,51 @@ public void testRenameTableAcrossSchema()
     }
 
     @Test
-    public void testDeleteAllTable()
+    public void verifySupportsDelete()
```

Comment:
This name is misleading to me. Maybe `testDeclaredSupportsDeleteBehavior`?

Reply:
Borrowed from BJCT - `verifySupports(RowLevel)DeleteDeclartion` (`verifySupportsJoinPushdownDeclaration`).